layers/README.md

```diff
@@ -0,0 +1,118 @@
+
+#### Core Modules
+| script | description |
+|:-------|:------------|
+| [filter](core/filter.py) | Implements a basic [filter object](https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv3.md#filter-object-properties). |
+| [gradient](core/gradient.py) | Implements a basic [gradient object](https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv3.md#gradient-object-properties). |
+| [layer](core/layer.py) | Provides an interface for interacting with core module's layer representation. A further breakdown can be found in the corresponding section below. |
+| [layout](core/layout.py) | Implements a basic [layout object](https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv3.md#layout-object-properties). |
+| [legenditem](core/legenditem.py) | Implements a basic [legenditem object](https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv3.md#legenditem-object-properties). |
+| [metadata](core/metadata.py) | Implements a basic [metadata object](https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv3.md#metadata-object-properties). |
+| [technique](core/technique.py) | Implements a basic [technique object](https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv3.md#technique-object-properties). |
+
+#### Manipulator Scripts
+| script | description |
+|:-------|:------------|
+| [layerops](manipulators/layerops.py) | Provides a means by which to combine multiple ATT&CK layer objects in customized ways. A further breakdown can be found in the corresponding section below. |
+
+## Layer
+The Layer class provides format validation and read/write capabilities to aid in working with ATT&CK Navigator Layers in python. It is the primary interface through which other Layer-related classes defined in the core module should be used. The Layer class API and a usage example are below.
+
+| method [x = Layer()]| description |
+|:-------|:------------|
+| x.load_input(_input_) | Loads an ATT&CK layer from either a dictionary or a string representation of a dictionary. |
+| x.load_file(_input_) | Loads an ATT&CK layer from a file location specified by the _input_. |
+| x.export_file(_input_) | Saves the current state of the loaded ATT&CK layer to a json file denoted by the _input_. |
+| x.get_dict() | Returns a representation of the current ATT&CK layer object as a dictionary. | 
```

Comment:
Sorry for having this thought so late, but I think we might want to rename these functions.

| before | after |
|:---|:---|
| load_input | break into `from_str` and `from_dict` |
| load_file | from_file |
| export_file | to_file |
| get_dict | to_dict |
| (new function) |  to_str |

I think this is more standardized and uses better IO terminology. Hopefully easier to remember for users.

Reply:
To_str should basically just be `json.dumps(self.to_dict())`